packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -91,6 +92,11 @@ class CupertinoSwitch extends StatefulWidget {
   /// Defaults to [CupertinoColors.activeGreen].
   final Color activeColor;
 
+  /// If set to [DragStartBehavior.start], switching drag behavior will
+  /// begin upon the drag gesture winning the arena. If set to
+  /// [DragStartBehavior.down] it will begin when a down event is first detected.
+  final DragStartBehavior dragStartBehavior;
```

Comment:
nit: documentation should be in this form:
```
/// Brief one-sentence description.
///
/// Explanation of each value...
```

also, grammar nit: your first sentence has a comma after the "if set to x", the second doesn't. We should definitely be consistent...